src/main/java/seedu/address/logic/commands/SetPathCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.logic.commands;
+
+import seedu.address.commons.core.ProfilePicturesFolder;
+import seedu.address.logic.commands.exceptions.CommandException;
+
+/**
+ * Sets the absolute path to the profile pictures folder that is residing in user's workspace.
+ */
+public class SetPathCommand extends UndoableCommand {
+
+    public static final String COMMAND_WORD = "setpath";
+    public static final String MESSAGE_SUCCESS = "Location to access profile pictures is now set!";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + "C:/Users/acer/Desktop/SE/profilepic/";
```

Comment:
same as above

Reply:
Also, this will cause a problem if the address book is used by many people on different computers because file path does not exist. Maybe there's a way to copy the picture from user's workspace into the app's folder?

Reply:
Yes, you are right. I just set up the basic functionality to ensure it is not breaking anything. Although this is trivial way to implement this, it is a stepping stone for my next PR, where I'm trying to retrieve the image and save it inside the app itself.